Packs/PhishingURL/Scripts/DBotPredictURLPhishing/DBotPredictURLPhishing.py

```diff
@@ -460,8 +460,11 @@ def extract_created_date(entry: dict):
 
 def weed_rasterize_errors(urls: list[str], res_rasterize: list[dict]):
     '''Remove the URLs that failed rasterization and return them.'''
-
-    error_idx = [i for (i, res) in enumerate(res_rasterize) if isinstance(res['Contents'], str)]
+    if len(urls) != len(res_rasterize):
+        demisto.debug(f'{res_rasterize=}')
+        raise DemistoException('Unexpected response from the "rasterise" command.'
+                               'Please make sure the Rasterize pack is up-to-date and functional')
+    error_idx = [i for (i, res) in enumerate(res_rasterize) if isinstance(res['Contents'], str)][::-1]
```

Comment:
this is very hard to read. lets for loop over the response and do return_results in a regular for